tmail-backend/apps/postgres/src/main/java/com/linagora/tmail/james/app/PostgresTmailServer.java

```diff
@@ -239,4 +241,11 @@ private static Module chooseRedisRateLimiterModule(PostgresTmailConfiguration co
         }
         return Modules.EMPTY_MODULE;
     }
+
+    private static Module chooseRspamdModule(PostgresTmailConfiguration configuration) {
```

Comment:
Do we have rspamd IT for postgres-app yet? Maybe `TMailHealthCheckIntegrationTests` is a good candidate?

Reply:
I tried but Rspamd require MessageIdManager 
So, I will drop chooseRspamdModule now,